src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Common/CustomMessageNames.cs

```diff
@@ -29,7 +30,6 @@ internal static class CustomMessageNames
     public const string RazorResolveCodeActionsEndpoint = "razor/resolveCodeActions";
     public const string RazorProvideHtmlColorPresentationEndpoint = "razor/provideHtmlColorPresentation";
     public const string RazorProvideHtmlDocumentColorEndpoint = "razor/provideHtmlDocumentColor";
-    public const string RazorPullDiagnosticEndpointName = "razor/pullDiagnostics";
```

Comment:
Previously VS and VS Code both used this method name, but to my surprise, with different response types. Not strictly an issue since the two systems never exist together, but I thought it would be easier to audit things in future if it wasn't confusing.